docs/support/troubleshooting.md

```diff
@@ -5,8 +5,9 @@
 This error happens when the commit associated with the last release cannot be found in the branch history.
 
 Multiple situation can cause this issue:
-- The package name configured in your `package.json` already exists and **semantic-release** obtains the last release of that package, which is unrelated to yours
```

Comment:
Can remove this change of syntax that is unrelated to this PR?

Reply:
Oh, looks like my editor made a formatting change i.e. changed `-` to `*` So, the copy is not actually removed.

Reply:
I don't understand what you mean... Please remove your modification so `-` stays `-`.

Reply:
I will revert. More specifically, I believe it might be prettier-atom that did the reformatting.